main.tf

```diff
@@ -1,14 +1,14 @@
 provider "aws" {
-     region  = "us-east-1"        
+  region = "us-east-1"
 }
 
 terraform {
-    required_providers {
-      aws = {
-        source = "hashicorp/aws"
-        version = "~> 4.16"
-      }
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
     }
+  }
 
-    required_version = ">= 1.2.0"
+  required_version = ">= 1.2.0"
 }
```

Comment:
Please add an extra new line to the end of this file so you don't see the ⛔ sign on github.